addons/mail/static/src/messaging/entity/message/message.js

```diff
@@ -371,6 +385,20 @@ function MessageFactory({ Entity }) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsNeedaction() {
+            const inbox = this.env.entities.Thread.find(
+                thread => thread.model === 'mail.box' && thread.id === 'inbox'
+            );
```

Comment:
inbox thread should probably be a singleton on messaging if not the case already

Reply:
Makes sense for all mailboxes btw. Will attempt that in my branch, don't think it's too hard to do.

Reply:
Ok for other mailboxes too.